doc/reference.conf

```
@@ -546,14 +546,20 @@ operator "god" {
 	privset = "admin";
 };
 
-/* connect {}: controls servers we connect to (OLD C:, N:, H:, L:) */
+/* connect {}: controls servers we connect with (OLD C:, N:, H:, L:).
+ *
+ * This configuration is used whether connections are incoming or
+ * outgoing.
+ */
 connect "irc.uplink.com" {
 	/* the name of the other server must go above. It should match the
 	 * other server's name in its serverinfo {} block, and does not
 	 * need to be an actual hostname. */
```

Gonna also fix this since I noticed it, because gross.

```suggestion
	 * need to be an actual hostname.
	 */
```